capreolus/reranker/tests/test_rerankers.py

```diff
@@ -596,3 +597,184 @@ def test_birch(dummy_index, tmpdir, tmpdir_as_cache, monkeypatch):
     reranker.trainer.train(
         reranker, train_dataset, Path(tmpdir) / "train", dev_dataset, Path(tmpdir) / "dev", benchmark.qrels, metric
     )
+
+
+# def test_parade_maxp(dummy_index, tmpdir, tmpdir_as_cache, monkeypatch):
```

Comment:
Should the tests be commented?

Reply:
Yeah I think so. The github action kills itself otherwise I think.